### PR TITLE
DCS-1436 added exitUrl to template variables for dps link fix in header

### PR DESCRIPTION
**Pull request by louisenorris**

### CODE REVIEW

**Review by connormaglynn — APPROVED** [review]
lgtm

**Review by andrewrlee — DISMISSED** [review]
LGTM

File: licences-specs/src/test/groovy/uk/gov/justice/digital/hmpps/licences/specs/common/WebsiteSpec.groovy [hunk 1]
```diff
@@ -63,11 +63,7 @@ class WebsiteSpec extends GebReportingSpec {
 
     then: 'I see health status OK'
 
-    def json = (
-      driver.pageSource
-        - ' xmlns="http://www.w3.org/1999/xhtml"'
-        - '<html><head></head><body><pre style="word-wrap: break-word; white-space: pre-wrap;">'
-    ) - '</pre></body></html>'
+    def json = (driver.pageSource.replaceAll("\\<.*?>", "")) - '</pre></body></html>'
```
Comment (andrewrlee) [inline licences-specs/src/test/groovy/uk/gov/justice/digital/hmpps/licences/specs/common/WebsiteSpec.groovy:66]: technically shouldnt need the`'</pre></body></html>'` either

Comment (louisenorris) [reply licences-specs/src/test/groovy/uk/gov/justice/digital/hmpps/licences/specs/common/WebsiteSpec.groovy:66]: Ah cool, will make this update now quickly

**Review by andrewrlee — APPROVED** [review]
lgtm

**Pull request merged**